models/tiktok_ads.yml

```diff
@@ -447,4 +447,73 @@ models:
       - name: daily_cpm
         description: The average amount of money you've spent per 1,000 impressions.
       - name: daily_ctr
-        description: The percentage of times people saw your ad and performed a click.
\ No newline at end of file
+        description: The percentage of times people saw your ad and performed a click.
+
+  - name: tiktok_ads__campaign_country_report
+    description: Each record in this table represents the daily performance of a campaign at the country/geographic region level.
```

Comment:
If this model is representing country and geographic regions, should we rename the model to be more generic, like `tiktok_ads__campaign_geographic_report`? Or is the plurality of data for `country_code` usually countries?

Reply:
Yeah the plurality of data for country_code is typically countries with a few outliers based on TikTok Ads API docs. We opted to call out the geographic regions in the description to highlight that there are some outliers, but they are not the majority. Additionally, since there are only a few outliers and not complete support for regions, I didn't want to name the model as such. 

Lastly, I would prefer the name of the model match the source name as closely as possible.